manof/image.py

```diff
@@ -1,5 +1,6 @@
 import os
 import pipes
+import json
```

Comment:
we use `simplejson` (updated more frequently), and in any case, you don't need to use it directly (see `Target.pprint_json()`)